firefox-ios/Client/Frontend/Fakespot/Views/FakespotActionFooterView.swift

```diff
@@ -95,9 +95,11 @@ public final class FakespotActionFooterView: UIView, ThemeApplicable {
     }
 
     // MARK: - ThemeApplicable
-    public func applyTheme(theme: Common.Theme) {
-        linkButton.setTitleColor(theme.colors.actionPrimary, for: .normal)
-        linkButton.setTitleColor(theme.colors.actionPrimaryHover, for: .highlighted)
-        titleLabel.textColor = theme.colors.textSecondary
+    public func applyTheme(theme: Theme) {
+        let colors = theme.colors
+        linkButton.applyTheme(theme: theme)
+        linkButton.setTitleColor(colors.actionPrimary, for: .normal)
```

Comment:
The title color doesn't have to be set anymore as the `LinkButton` has the correct styles already.